tests/test_polynomial.py

```diff
@@ -0,0 +1,60 @@
+from honeybadgermpc.field import GF
+import honeybadgermpc.polynomial as polynomial
+from random import randint
+
+Field = GF(0x73eda753299d7d483339d80809a1d80553bda402fffe5bfeffffffff00000001)
+Poly = polynomial.polynomialsOver(Field)
```

Comment:
We can make use of the following fixtures: https://github.com/initc3/HoneyBadgerMPC/blob/dev/tests/conftest.py#L27-L38

Reply:
p.s.: If using the fixtures, you'll need to invoke them for each test function: e.g.:

```python
def test_poly_eval_at_k(GaloisField, Polynomial):
    poly1 = Polynomial([0, 1])  # y = x
    # ...
    coeffs = [randint(0, GaloisField.modulus-1) for i in range(d)]
    # ...
```